docs/api-spec.md

```
@@ -0,0 +1,426 @@
+  - `ModeListItem`: `{ mode_id, name, icon, is_active }`
+
+### POST `/modes`
+
+- **Request Body** (`ModeCreateRequest`)
+
+  | 필드                                                                        | 타입                        | 필수 |
+  | --------------------------------------------------------------------------- | --------------------------- | ---- |
+  | `name`                                                                      | string                      | ✅   |
+  | `icon`                                                                      | string                      | ✅   |
+  | `sounds`                                                                    | array&lt;ModeSoundInput&gt; | ✅   |
+  | - `ModeSoundInput`: `{ sound_id: integer (required), name?: string\|null }` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)